datadog_checks_dev/datadog_checks/dev/tooling/commands/env/test.py

```diff
@@ -0,0 +1,69 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import click
+
+from ...testing import get_tox_envs
+from ..console import CONTEXT_SETTINGS, echo_info, echo_warning
+from ..test import test as test_command
+from .start import start
+from .stop import stop
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Test an environment')
+@click.argument('checks', nargs=-1)
+@click.option(
+    '--agent',
+    '-a',
+    default='6',
+    help=(
+        'The agent build to use e.g. a Docker image like `datadog/agent:6.5.2`. For '
+        'Docker environments you can use an integer corresponding to fields in the '
+        'config (agent5, agent6, etc.)'
+    ),
+)
+@click.option('--dev/--prod', help='Whether to use the latest version of a check or what is shipped')
+@click.option('--base', is_flag=True, help='Whether to use the latest version of the base check or what is shipped')
+@click.option(
+    '--env-vars',
+    '-e',
+    multiple=True,
+    help=(
+        'ENV Variable that should be passed to the Agent container. '
+        'Ex: -e DD_URL=app.datadoghq.com -e DD_API_KEY=123456'
+    ),
+)
+@click.option('--clean', '-c', is_flag=True, help='Execute setup and tear down actions')
```

Comment:
maybe sth like `--new-env` clean has already a lot of meaning from build scripts and it sounds confusing

Reply:
+1 for `--new-env`.